src/net.cpp

```diff
@@ -2806,8 +2806,8 @@ void CConnman::RecordBytesSent(uint64_t bytes)
     LOCK(cs_totalBytesSent);
     nTotalBytesSent += bytes;
 
-    uint64_t now = GetTime();
-    if (nMaxOutboundCycleStartTime + nMaxOutboundTimeframe < now)
+    auto now = GetTime<std::chrono::seconds>();
```

Comment:
832fc10e9e45abf9 const and maybe explicit typing
```suggestion
    const std::chrono::seconds now{GetTime<std::chrono::seconds>()};
```

Reply:
Having `std::chrono::seconds` twice on the same line is pretty verbose. I'd like to think we can just use `auto` here because the type is clear from the call to GetTime<>().

Reply:
Yes, the reason for the comment here was constness.

Reply:
Note that existing code in the same file specifies the type explicitly, line 1646: `const std::chrono::seconds now = GetTime<std::chrono::seconds>();`